### DISCUSSION TITLE
Can we add a relationship diagram between roles？

### DISCUSSION THREAD

**Discussion by lianglivi** [vkbo/novelWriter#1952] [Q&A]
Can you add a feature to build relationships between characters, represented in the form of charts, to facilitate inspiration for creators, and can also be used on some combined items. I really hope to have this feature, it's really necessary.
![bb625571bc2ae487ea6c6c7eaa09af09](https://github.com/vkbo/novelWriter/assets/169472920/3ae131c3-0eb1-4b69-9021-194c3b978299)

**Comment by vkbo** [comment]
This may be a useful feature. You can already cross-link characters, but there is no chart.

Please make a feature request under "Issues".
